ipv6/miredo/patches/101-client-hook.patch

```diff
@@ -0,0 +1,12 @@
+--- a/misc/client-hook.iproute
++++ b/misc/client-hook.iproute
+@@ -5,7 +5,7 @@
+ # Distributed under the terms of the GNU General Public License version 2.
+ 
+ # Linux iproute2 path:
+-IP="/sbin/ip"
++IP="/usr/sbin/ip"
```

Comment:
Could you make it use ip without an absolute path here or does the application does not have the PATH when executing this?

Reply:
I checked, but absolute path is required here.
(Using a relative one would probably be a potential security issue anyway.)